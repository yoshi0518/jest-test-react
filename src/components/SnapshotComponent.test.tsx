import SnapshotComponent from "./SnapshotComponent";
import { render } from "@testing-library/react";

it("SnapshotTest", () => {
  const { container } = render(<SnapshotComponent text="React" />);
  expect(container).toMatchSnapshot();
});
