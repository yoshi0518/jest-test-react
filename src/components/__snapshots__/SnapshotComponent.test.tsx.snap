// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`SnapshotTest 1`] = `
<div>
  <div>
    <p>
      React
    </p>
  </div>
</div>
`;
